<?xml version="1.0"?>

<launch>

    <node name="hybrid_control" pkg="output_controller" type="output_control.py" output="screen">
        <param name="name" value="hybrid"/>
        <param name="arduino_case" value="5"/>
    </node>

    <node name="force_control" pkg="output_controller" type="force_control.py" output="screen">
        <param name="arduino_case" value="4"/>
    </node>

</launch>

